ext/drx/drx.c

```diff
@@ -2540,97 +2556,78 @@ static bool
 drx_try_to_detect_scatter_gather_sequence(
```

Comment:
I thought the names were changed?

Reply:
I had changed the drx_restore_state_for_avx[512|2]_[gather|scatter] names. I left this one because I intentionally left the top function. My thinking was it should still be caled "try_" in order to reflect that it is "trying" to detect the code. I think this should still be in the same somehow. What about drx_try_restore_state_for_scatter_gather?

Reply:
IMHO a function named "try_to_detect" is promising to be a const non-mutating function: it just returns whether it detecting something.

> What about drx_try_restore_state_for_scatter_gather?

OK, or drx_restore_state_if_scatter_gather.

Reply:
I wanted to catch somehow that it is not a bullet proof restore, while prob. sufficiently accurate, it still is an approximation. I can catch this just by adding a comment to the function.

Reply:
Done.